android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboardPickerAdapter.java

```diff
@@ -47,10 +47,26 @@ public KMKeyboardPickerAdapter(final Context context, Dataset.Keyboards adapter)
         List<HashMap<String, String>> kbdMapList = KeyboardPickerActivity.getKeyboardsList(context);
         List<Keyboard> kbdList = new ArrayList<>(kbdMapList.size());
 
-        for(HashMap<String, String> kbdMap: kbdMapList) {
+        int kbdListSize = (kbdMapList != null) ? kbdMapList.size() : 1;
+        List<Keyboard> kbdList = new ArrayList<>(kbdListSize);
+
+        if (kbdMapList != null) {
+          for (HashMap<String, String> kbdMap : kbdMapList) {
+            kbdList.add(new Keyboard(kbdMap));
+          }
+        } else {
+          // Couldn't access keyboard list so just return default keyboard
```

Comment:
This kinda masks the symptoms of the issue. Why can't it access the keyboard list?